Mathlib/Algebra/Category/Ring/Constructions.lean

```diff
@@ -103,6 +104,17 @@ def pushoutCoconeIsColimit : Limits.IsColimit (pushoutCocone R A B) :=
     rw [← h.map_mul, Algebra.TensorProduct.tmul_mul_tmul, mul_one, one_mul]
     rfl
 
+lemma nontrivial_of_isPushout_of_isField {A B C D : CommRingCat.{u}}
```

Comment:
As this increases significantly the imports, this lemma should be moved to another file. (Alternatively, you could use that more generally the tensor product of two nontrivial vector spaces is nontrivial, which follows from the existence of nontrival linear forms.)

Reply:
Okay I think I fixed it by making a new file. Note that this lemma uses the more general statement that you mention, just by realizing that free modules are faithfully flat.